kaldb/src/main/proto/metadata.proto

```diff
@@ -109,3 +109,28 @@ message RecoveryTaskMetadata {
   // Created timestamp
   int64 created_time_epoch_ms = 5;
 }
+
+message ServiceMetadata {
```

Comment:
While it's obvious now, in the past it was harder to understand what these structs did at a later time. So, add a comment stating that this struct captures service ownership information, the throughput allocated to the service and the owner. 

Also, define what the partition_configs field would contain.

Reply:
Comments added to top-level objects, and improved description of the `parition_configs`.